### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by vinnywright** [zaps166/QMPlay2#265]
HI 
first let me say I love this app , that said some people I try to get to use it are put off by it not having easily found help , as in click the "help" in the menu and find some user documentation on how to use it .
if their is something like this I have not found , where is it , and/or can it be done .

I love QMplay2 myself and have no problems with it for what I do with it , but some kind of help fie or page would be useful for some .
VINNY

**Comment by zaps166** [comment]
Good idea. The documentation can be a Wiki on Github instead of huge readme (and link from help menu), but there must be a person who can write it. Personally I hate writing docs :smile:

**Comment by vinnywright** [comment]
lol , I understand .
I would love to contribute in some way as I love the player and have been using it for some time now the installed .deb's and now the appimage on Kubuntu and Neon-Testing 

however anything I tried to write would be vary basic and will take some time as I have little of it , time that is .
I also I have no experience with "wiki" pages , BUT could try if it could be kept in the dark until finished and approved by you .
I also think screen shots and screen captures (youtube links) could be good (I can do those) 

VINNY
